locales/mr/translation.json

```
@@ -0,0 +1,256 @@
+        "copy": "कॉपी करा",
+        "could-not-be-loaded": "लोड केले जाऊ शकले नाही",
+        "cut": "कट",
+        "database-export": "डेटाबेस निर्यात",
+        "database-imported": "डेटाबेस आयात केला",
+        "database-migrated": "डेटाबेस स्थलांतर",
+        "database-was-exported": "ठीक आहे, डेटाबेस निर्यात झाला.",
+        "edit": "सुधारणे",
+        "exit": "बाहेर पडा",
+        "export-database": "डेटाबेस निर्यात करा",
+        "export-db-to-file": "फाईलमध्ये डेटाबेस निर्यात करा",
+        "failed-entries": "नोंदी अयशस्वी",
+        "failed-migrating": "स्थलांतर करण्यात अयशस्वी",
+        "help": "मदत करा",
+        "import": "आयात करा",
+        "import-database": "आयात डेटाबेस",
+        "import-db-from-file": "फाईलमधून डेटाबेस आयात करा",
+        "import-successful": "होय! आयात यशस्वी!",
+        "menu": "मेनू",
+        "migrate-calendar-to-flexible": "निश्चित कॅलेंडर डेटाबेस लवचिकमध्ये स्थलांतरित करा",
+        "migrate-to-flexible-calendar": "लवचिक कॅलेंडरमध्ये स्थलांतर करा",
+        "migration-successful": "होय! स्थलांतर यशस्वी!",
+        "no-thanks": "नको धन्यवाद",
+        "ok": "ठीक आहे",
+        "paste": "पेस्ट करा",
+        "preferences": "प्राधान्ये",
+        "punch-time": "पंच वेळ",
+        "punched-time": "मुक्कामी वेळ",
+        "quit": "सोडा",
+        "reload": "रीलोड करा",
+        "select-all": "सर्व निवडा",
+        "send-feedback": "अभिप्राय पाठवा",
+        "should-migrate-to-flexible": "आपले लवचिक कॅलेंडर रिक्त आहे. आपण आपल्या लवचिकमध्ये विद्यमान निश्चित कॅलेंडर डेटाबेसचे स्थानांतरन सुरू करू इच्छिता?",
+        "show-app": "अ‍ॅप दर्शवा",
+        "something-went-wrong": "काहीतरी चूक घडली :(",
+        "toggle-dev-tools": "विकसक साधने टॉगल करा",
+        "ttl-github": "टीटीएल गिटहब",
+        "view": "पहा",
+        "workday-waiver-manager": "वर्क डे माफी व्यवस्थापक",
+        "yes-please": "होय करा"
+    },
+    "$FlexibleDayCalendar": {
+        "current-day": "चालू दिवस जा",
+        "day-done-balance": "आज सर्व केले. दिवसाची शिल्लक:",
+        "day-total": "दिवस एकूण",
+        "entry": "प्रवेश",
+        "entry-removal-confirmation": "आपली खात्री आहे की आपण शेवटची प्रविष्टी पंक्ती हटवू इच्छिता?",
+        "leave-by": "आपण याद्वारे सोडले पाहिजे:",
+        "month-balance": "महिना शिल्लक",
+        "month-balance-title": "या महिन्यासाठी आज पर्यंत शिल्लक ठेवा. सकारात्मक संतुलनाचा अर्थ असा आहे की आपल्याला आज (किंवा महिना उर्वरित) काम करण्याची आवश्यकता नाही.",
+        "next-day": "पुढचा दिवस",
+        "no": "नाही",
+        "not-a-working-day": "कामकाजाचा दिवस नाही",
+        "overall-balance": "एकूणच शिल्लक",
+        "overall-balance-title": "महिन्याच्या शेवटी किंवा चालू दिवसापर्यंत एकंदरीत शिल्लक",
+        "previous-day": "आदल्या दिवशी",
+        "remove-entry": "प्रविष्टी काढा",
+        "switch-view": "स्विच व्ह्यू",
+        "time-to-leave": "सोडण्याची वेळ",
```

Time to Leave is the name of the app, because of this we are not translating it:

```suggestion
        "time-to-leave": "Time to Leave",
```